stages/org.osbuild.rpm

```diff
@@ -92,15 +136,15 @@ def main(tree, sources, options):
     subprocess.run(["/bin/sh", "-c", script], check=True)
 
     with tempfile.NamedTemporaryFile(prefix="manifest.", mode='w') as manifest:
-        manifest.writelines(p+'\n' for p in packages)
+        manifest.writelines(c+'\n' for c in checksums)
         manifest.flush()
         subprocess.run([
             "rpm",
             "--verbose",
             "--root", tree,
-            # Make rpm require valid signatures & digests on packages.
-            # (see /usr/lib/rpm/macros for more info)
-            "--define", "_pkgverify_level all",
+            # The content hash of the rpms has been verified, default to not
+            # verifying again (see /usr/lib/rpm/macros for more info)
+            "--define", "_pkgverify_level none",
```

Comment:
That is the default, no? I think we can drop the directive.

Reply:
```
# Enforced package verification level
# all           require valid digest(s) and signature(s)
# signature     require valid signature(s)
# digest        require valid digest(s)
# none          traditional rpm behavior, nothing required
%_pkgverify_level digest
```
The traditional behaviour is of course not the default. I'd keep it, just to avoid stuff moving under us.

Reply:
:cool: